core_contracts/loans/loans/positions.py

```diff
@@ -43,8 +43,11 @@ def __getitem__(self, _symbol: str) -> int:
             revert(f'{TAG}: {_symbol} is not a supported asset on Balanced.')
 
     def __setitem__(self, key: str, value: int):
-        day = self.check_snap()
-        self.assets[day][key] = value
+        if self._loans.getDay() < self._loans._continuous_reward_day.get():
+            day = self.check_snap()
+            self.assets[day][key] = value
+        else:
+            self.assets[self.snaps[-1]][key] = value
```

Comment:
@Suyog007 - I thought we were going to remove the snaps index and make a dew data structure so all collateral types would be treated the same. Otherwise the structure will always have that unused key and extra level of depth.

Reply:
Actually , loans migration is implemented on another branch and it is yet to be merged with contiuous-integration branch.
https://github.com/balancednetwork/balanced-contracts/blob/loans-data-migration/core_contracts/loans/loans/positions.py#L60